doc/api/fs.md

```diff
@@ -1180,8 +1180,9 @@ to be notified of filesystem changes.
 
 If the underlying functionality is not available for some reason, then
 `fs.watch` will not be able to function.  For example, watching files or
-directories on network file systems (NFS, SMB, etc.) often doesn't work
-reliably or at all.
+directories on network file systems (NFS, SMB, etc.) and on exposed host file
```

Comment:
s/and on/or/?  I suspect most programmers are like me and interpret it boolean logic-style.